datahub-web-react/src/graphql/schemaBlame.graphql

```diff
@@ -0,0 +1,46 @@
+query getSchemaBlame($input: GetSchemaBlameInput!) {
+    getSchemaBlame(input: $input) {
+        latestVersion {
+            semanticVersion
+            semanticVersionTimestamp
+            versionStamp
```

Comment:
qq: what's the difference between semanticVersion and versionStamp?

Reply:
Yes, versionStamp is a string that looks something like `"browsePaths:0;dataPlatformInstance:0;datasetKey:0;editableSchemaMetadata:0;institutionalMemory:0;ownership:0;schemaMetadata:1;upstreamLineage:0"`